proto/auth/service.proto

```diff
@@ -63,35 +36,27 @@ message GetAuthenticationURLResponse {
   string url = 1;
 }
 
-message ExchangeBucketeerTokenRequest {
+message ExchangeTokenRequest {
   string code = 1;
   string redirect_url = 2;
   AuthType type = 3;
 }
 
-message ExchangeBucketeerTokenResponse {
+message ExchangeTokenResponse {
   Token token = 1;
 }
 
-message RefreshBucketeerTokenRequest {
+message RefreshTokenRequest {
   string refresh_token = 1;
-  string redirect_url = 2;
-  AuthType type = 3;
```

Comment:
We don't need to validate these two fields because the refresh token is generated on the Bucketeer server.